packages/webdriver-utils/src/providers/automateProvider.js

```diff
@@ -123,15 +123,16 @@ export default class AutomateProvider extends GenericProvider {
     });
   }
 
-  async getTiles(headerHeight, footerHeight, fullscreen) {
+  async getTiles(fullscreen) {
     if (!this.driver) throw new Error('Driver is null, please initialize driver with createDriver().');
     log.debug('Starting actual screenshotting phase');
     const dpr = await this.metaData.devicePixelRatio();
+    this.options.version = 'v2';
```

Comment:
```suggestion
    this.options.version = 'v2';
```

can modification of `options` moved to a single place? [ `percyAutomateRequestHandler` check this function under core/utils ]

Reply:
Also spec is pending for same.